react-common/components/controls/Feedback/configs.ts

```diff
@@ -0,0 +1,57 @@
+export const appId = 50315;
+export const feedbackFrameUrl = 'https://admin-ignite.microsoft.com';
+
+export const baseConfig: any = {
+    feedbackUiType: "NoSurface",
+    hostPlatform: "IFrame",
+    isDisplayed: true,
+    isEmailCollectionEnabled: false, // to enable email collection
+    isFileUploadEnabled: false, // to enable file upload function
+    isScreenshotEnabled: false, // to enable screenshot
+    isScreenRecordingEnabled: false, // to enable screen recording
+    invokeOnDismissOnEsc: false,
+    isFeedbackForumEnabled: false,
+    isMyFeedbackEnabled: false,
+    isThankYouPageDisabled: false,
+}
+
+export const ratingFeedbackConfig: any = {
+    ...baseConfig,
+    initialFeedbackType: "Unclassified",
+    scenarioConfig: {
+      isScenarioEnabled: true,
+      scenarioType: "Custom",
+      questionDetails: {
+        questionUiType: "Rating",
+        questionInstruction: {
```

Comment:
all of these question strings should be wrapped in `lf`. also, instead of declaring this as a variable you should create a function that returns it. otherwise, the language might not have been set before this file is imported

Reply:
any string that is shown to the user should be localized. strings that are just settings, like the "Rating" above shouldn't be.

Reply:
I actually think this is something OCV will do on their end.. I'd have to ask how they do it, but I might want to leave as is for now. 
![image](https://github.com/user-attachments/assets/5f32a507-e541-4e87-b13f-d0e55de1cff4)

Reply:
i don't think they could be translating it on the fly... looks like you need to pass the parameter both as a localized and non localized version. i bet that the english param is just what shows up in whatever UI they have for looking at the feedback folks send in.

Reply:
Ah, I see. I will update it, then, thanks!